server.js

```diff
@@ -40,13 +40,22 @@ app.post(routes.loginRoute, (req, res) => {
         .catch(err => console.error(err));
 });
 
+app.post('/account', (req, res) => {
+    console.log('here');
+    if(req.body){
+        console.log(req.body);
+        res.status(200).json({status: 'ok'});
+    } else {
+        res.status(400).json({status: 'not ok'});
+    }
+});
+
 // route to check the token for validity
 app.get(routes.checkTokenRoute, (req, res) => {
-    console.log(req.headers);
     if(req.headers){
         jwt.verify(req.headers.authorization.split(' ')[1], secretConfig.secret, function(err, decoded){
-            console.log(decoded);
-            if(err){
+            let date = new Date().getTime();
```

Comment:
use const except of let if you do not change date later

Reply:
done